### PR TITLE
Migrated compact to GPL

### PR DESCRIPTION
**Pull request by minecraft7771**
- Fixes https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/13962
- Uses GPL to find fusion recipes now.
- Deleted most contents of the gtpp fusion base class since it was mostly a copy of the regular one

### CODE REVIEW

File: src/main/java/goodgenerator/blocks/tileEntity/base/LargeFusionComputer.java [hunk 1]
```diff
@@ -146,6 +148,15 @@ public boolean allowCoverOnSide(ForgeDirection side, GT_ItemStack aStack) {
         return side != getBaseMetaTileEntity().getFrontFacing();
     }
 
+    @Override
+    public void loadNBTData(NBTTagCompound aNBT) {
+        super.loadNBTData(aNBT);
+        // Migration code
+        if (lEUt > 0) {
+            lEUt = -lEUt;
```
Comment (BlueWeabo) [inline src/main/java/goodgenerator/blocks/tileEntity/base/LargeFusionComputer.java:156]: Shouldn't you also migrate mEUt as it was using that before hand?

Comment (minecraft7771) [reply src/main/java/goodgenerator/blocks/tileEntity/base/LargeFusionComputer.java:156]: No that is handled by TT. It loads lEUt with the key "mEUt"

File: src/main/java/goodgenerator/blocks/tileEntity/base/LargeFusionComputer.java [hunk 2]
```diff
@@ -388,101 +399,81 @@ public void onMachineBlockUpdate() {
...
-            if ((tRecipe == null && !mRunningOnLoad) || (tRecipe != null && (maxEUStore() < tRecipe.mSpecialValue))) {
-                turnCasingActive(false);
-                this.mLastRecipe = null;
-                return CheckRecipeResultRegistry.NO_RECIPE;
+            @NotNull
+            @Override
+            protected GT_OverclockCalculator createOverclockCalculator(@NotNull GT_Recipe recipe,
+                    @NotNull GT_ParallelHelper helper) {
+                return super.createOverclockCalculator(recipe, helper)
+                        .limitOverclockCount(overclock(recipe.mSpecialValue));
```
Comment (BlueWeabo) [inline src/main/java/goodgenerator/blocks/tileEntity/base/LargeFusionComputer.java:450]: Any reason you aren't setting the OC to be 4/4 when its an mk4/5?

Comment (minecraft7771) [reply src/main/java/goodgenerator/blocks/tileEntity/base/LargeFusionComputer.java:450]: I set it in the PP base class.

Comment (BlueWeabo) [reply src/main/java/goodgenerator/blocks/tileEntity/base/LargeFusionComputer.java:450]: missed that one cus i only saw red there :trollface:

**Review by BlueWeabo — APPROVED** [review]
Fusion of repeated code

**Pull request merged**